readme.md

```diff
@@ -5,35 +5,56 @@
 ## Conteúdo
 
 - [Bases de Dados](#bases-de-dados)
+- [Biblioteca](#biblioteca)
 - [Livros](#livros)
 - [Podcasts](#podcasts)
 - [Projectos](#projectos)
 - [Redes Sociais](#redes-sociais)
-- [Vídeos](#videos)
+- [Videos](#videos)
 - [Diversos](#diversos)
 
 ## Bases de Dados
 
 - [PORDATA](https://www.pordata.pt/)
 
-## Livros
+## Biblioteca
+
+### Comece aqui a Introdução à Política
+
+- [Introdução à Ciência Política: Teorias, Métodos e Temáticas](https://www.goodreads.com/book/show/18397436-introdu-o-ci-ncia-pol-tica)
+- [Curso de Ciência Política](https://www.wook.pt/livro/curso-de-ciencia-politica-gianfranco-pasquino/4042597)
+- [Reinventar a Democracia](https://www.goodreads.com/book/show/25862711-reinventar-a-democracia?ac=1&from_search=true&qid=0n8VaBkfd3&rank=1)
+- [História da Filosofia Política](https://www.wook.pt/livro/historia-da-filosofia-politica-joao-cardoso-rosas/24190059)
+- [Manual da Teoria Política Aplicada](https://www.wook.pt/livro/manual-de-teoria-politica-aplicada-orlando-vitorino/4299597)
+
+#### Liberalismo
+
+-[Biblioteca +Mais Liberdade] (https://maisliberdade.pt/biblioteca/)
+
+
+### Livros Diversos em Português
 
 - [A força das Ideias: Histórias de uma eleição](https://www.goodreads.com/book/show/56313762-a-for-a-das-ideias)
 - [A Grande Escolha - Mundo Global ou Países Fechados?](https://www.goodreads.com/book/show/55606199-a-grande-escolha---mundo-global-ou-pa-ses-fechados)
 - [A República à Deriva](https://www.goodreads.com/book/show/55951697-a-rep-blica-deriva)
 - [Descentralização e Poder Local em Portugal](https://www.goodreads.com/book/show/56924620-descentraliza-o-e-poder-local-em-portugal)
 - [Esquerda e Direita: Guia Histórico Para o Século XXI](https://www.goodreads.com/book/show/25739364-esquerda-e-direita)
-- [Introdução à Ciência Política: Teorias, Métodos e Temáticas](https://www.goodreads.com/book/show/18397436-introdu-o-ci-ncia-pol-tica)
 - [O Economista Insurgente](https://www.goodreads.com/book/show/22665203-o-economista-insurgente)
 - [O Pequeno Livro Negro da Corrupção](https://www.goodreads.com/book/show/55252675-o-pequeno-livro-negro-da-corrup-o)
 - [Três retratos: Salazar, Cunhal, Soares](https://www.goodreads.com/book/show/54269622-tr-s-retratos)
 
 ## Podcasts
 
+#### Portugal 
```

Comment:
Tem um "#" a mais, deveriam ser 3

Reply:
hmm, percebo-te. Faz sentido. No entanto se for necessário adicionar mais subcategorias como "Geopolítica", "Ciência", convém meter ####

Reply:
no momento em que adicionarmos mais subcategorias podemos alterar, por agora esta é uma subcategoria de "Podcasts" que tem "##", por isso este tem 3

Mais informação aqui: https://www.markdownguide.org/basic-syntax/